### PR TITLE
Edits for Ansible lab, Python virtual environment

### PR DESCRIPTION
**Pull request by annegentle**
This PR uses the `> Note:` formatting and I will need to do the same to other .md files in this repo if you like this format. I'm not attached to either but of course prefer consistency.

Also, please look for an inline comment about a cross reference as I wasn't sure what the intent is.

### CODE REVIEW

File: labs/02-ansible-05-home-lab-ansible/4.md [hunk 1]
```diff
@@ -1,80 +1,91 @@
...
+Virtual environments provide many useful features. As a good habit with Python, use a virtual environment for each project. This lab shows the creation and use of a virtual environment for this lab, added to a subdirectory named AN01 (shorthand for Ansible lab 1), as shown in the next figure.
 
 ![alt text](/posts/files/02-ansible-05-home-lab-ansible/assets/images/desktop-5-12.png)
 
-You may of course choose to not use a virtual environment, but the lab steps assume the use of the virtual environment as shown here. Check out the second lab in the “[Home Lab: Setting up Your Desktop OS for Network Programmability](https://learninglabs.cisco.com/modules/home-lab-desktop)” Learning Module, for the lab about PIP and virtual environments, for more background info.
+You may choose not to use a virtual environment, but the lab steps assume the use of the virtual environment. Check out the second lab in [Home Lab: Setting up Your Desktop OS for Network Programmability](https://learninglabs.cisco.com/modules/home-lab-desktop) for more background info about PIP and virtual environments.
```
Comment (annegentle) [inline labs/02-ansible-05-home-lab-ansible/4.md]: I'm not sure if I'm already in the second lab? Aren't we in the lab about PIP and virtual environments?

**Comment by annegentle** [comment]
Ah, one additional note. I'm trying to revise to avoid the use of "page" and instead use "section." My thinking is that these may render as entire large HTML pages or as book pages some day, so it's preferable to use generic terms.

File: labs/02-ansible-05-home-lab-ansible/4.md (hunk 1 above)
Comment (WendellOdom) [reply labs/02-ansible-05-home-lab-ansible/4.md]: Anne, on your literal questions: you are not in the 2nd lab, and it's not a lab about PIP and virtual environments. You're in module 2 lab 5, which is about Ansible. 

You may want to check my comment with Paul's view of file naming. When we discussed how to name the .md files, he told me the standard, but it sounded like that standard might change. 

The standard was for the JSON file describing one lab:: 
modulenum-shortname-labnum-longname.json
e.g., I created two modules with several labs each, with each lab containing 8-12 web pages. So,
02-ansible-05-home-lab-ansible.json defines the files used with my module 2 lab 5. That naming is module 02, short name Ansible, lab 05 in that module, followed by a longer version of the name (does not have to literally match the published lab name). That JSON file lists 12 .MD files, with names like 1.md, 2.md, etc. Those files are the individual web pages in the lab, numbered 1 through N. So, if I get your question, it looks like you're editing 4.md, but it's the 4.md in the directory associated with 02-ansible-05-home-lab-ansible.json. 
For context, see https://learninglabs.cisco.com:8867/modules/home-lab-desktop, which lists the 5 lab exercises in my module 2.

**Comment by WendellOdom** [comment]
Reviewed copy edits; all are accepted.

**Pull request merged**